locksmith/scripts/fulfillment_prepare

```diff
@@ -1,16 +1,18 @@
 #!/usr/bin/env node
 
-require('dotenv').config()
+const path = require('path')
```

Comment:
Since we are copying this bit of code again here lets extract it to `utils` and give it a reasonable name